lib/api/element-commands/clickAndHold.js

```diff
@@ -42,7 +42,7 @@ const BaseElementCommand = require('./_baseElementCommand.js');
  * @param {string} selector The CSS/Xpath selector used to locate the element.
  * @param {function} [callback] Optional callback function to be called when the command finishes.
  * @api protocol.elementinteraction
- * @link /#dfn-element-clickAndHold
+ * @link https://www.selenium.dev/documentation/webdriver/actions_api/mouse/#click-and-hold
```

Comment:
Can you move all the links that are not related to the WebDriver Spec to `@see` tag?

Reply:
Sure